google/datalab/storage/_bucket.py

```diff
@@ -235,18 +235,6 @@ def contains(self, name):
       raise e
     return True
 
-  def create(self, name):
-    """Creates a new bucket.
-
-    Args:
-      name: a unique name for the new bucket.
-    Returns:
-      The newly created bucket.
-    Raises:
-      Exception if there was an error creating the bucket.
-    """
-    return Bucket(name, context=self._context).create(self._project_id)
```

Comment:
We expect users to use Bucket().create() directly now, so they send the context into that.